src/ducks/Alert/utils.js

```diff
@@ -175,3 +182,266 @@ export function splitStr (str) {
 
   return { firstWord, rest }
 }
+
+export function validateFormSteps ({
```

Comment:
Extract each case inside validator function, e.g. `validateAssetSteps()` and etc